spark-lineage/src/main/java/com/linkedin/datahub/lineage/consumer/impl/McpEmitter.java

```diff
@@ -13,18 +13,19 @@
 import com.linkedin.datahub.lineage.spark.model.LineageEvent;
 import com.linkedin.mxe.MetadataChangeProposal;
 
+import datahub.client.RestEmitter;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
-public class MCPEmitter implements LineageConsumer {
+public class McpEmitter implements LineageConsumer {
 
-  private static final String GMS_URL_KEY = "spark.datahub.lineage.mcpEmitter.gmsUrl";
+  private static final String GMS_URL_KEY = "spark.datahub.rest.server";
   private static final String SENTINEL = "moot";
 
-  private ConcurrentHashMap<String, RESTEmitter> singleton = new ConcurrentHashMap<>();
+  private ConcurrentHashMap<String, RestEmitter> singleton = new ConcurrentHashMap<>();
 
   private void emit(List<MetadataChangeProposal> mcps) {
-    RESTEmitter emitter = emitter();
+    RestEmitter emitter = emitter();
```

Comment:
We should ideally use a batch emit for this. @aseembansal-gogo is working on that API

Reply:
Please point me to the API documentation, will integrate it.